src/pages/lists/ListPage.tsx

```diff
@@ -38,6 +41,22 @@ export const ListPage: React.FC = () => {
     activeFilters,
     appliedFilters
   } = useFilters(filterConfig);
+  const { showSuccessMessage } = useMessages();
+
+  const updatedListsData = useLists([], [], undefined, undefined, updatedAsOf);
+  const updatedListsContent = updatedListsData?.listsData?.content;
+
+  if (updatedListsContent) {
```

Comment:
We can move all this logic in to useLists or create separate hook

Reply:
Moved into new `useListsFetchedSinceTimestamp()` hook